taxonomies/categories.txt

```diff
@@ -26063,6 +26174,18 @@ fr:Maasdam
 nl:Maasdam
 wikidata:en:Q3273732
 oqali_family:en: fr:Fromages - Maasdam
+ciqual_food_code:en:12705
+ciqual_food_name:en:Firm cheese, around 14% fat, Maasdam-type cheese, reduced fat
+ciqual_food_name:fr:Fromage à pâte ferme environ 14% MG type Masdaam à teneur réduite en MG
+
+<en:Uncooked pressed cheeses
+<en:Cow cheeses
+<en:Cheeses of the Netherlands
+en:Firm cheese around 27% fat Maasdam-type cheese
```

Comment:
We should apply the ciqual code to the Maasdam category.

Reply:
maasdam has another CIQUAL code (at 14% fat)